public/push-analytics.json

```diff
@@ -53,6 +89,12 @@
             "htmlSelector": "body",
             "cssSelector": "style"
         },
+        {
+            "dashboardItemProperty": "visualization.type",
+            "value": "OUTLIER_TABLE",
+            "strategy": "interceptResponse",
+            "urlGlob": "**/analytics/outlierDetection?displayProperty=NAME&**"
```

Comment:
This argument might have a different value depending on some settings.
In that case this won't match, perhaps remove the value or the entire query string?